### DISCUSSION TITLE
How to custom data on swagger documentation

### DISCUSSION THREAD

**Discussion by MakFly** [hussein4alaa/laravel-g4t-swagger-auto-generate#58] [General]
Hello, i have this : 

![image](https://github.com/hussein4alaa/laravel-g4t-swagger-auto-generate/assets/6107225/6de0fbf9-9ac8-4b4c-b1e0-1cadf6aeee10)

this is full custom api laravel , i havent use apiResource or rules request.

So how do I get custom parameters settings in the swagger?

**Comment by hussein4alaa** [comment]
Hello @MakFly! 
You should use request validation rules to get custom parameter in Swagger for your Laravel API.

**Comment by MakFly** [comment]
hum ok , i see , i go to try

**Reply by MakFly** [reply]
Thanks its working ! , but now how to hide a route on swagger if we wish ?
